contracts/OlasMech.sol

```diff
@@ -214,20 +223,18 @@ abstract contract OlasMech is Mech, IErrorsMech, ImmutableStorage {
         }
 
         // Check for max delivery rate compared to requested one
-        if (maxDeliveryRate > mechDelivery.deliveryRate) {
+        if (maxDeliveryRate < mechDelivery.deliveryRate) {
```

Comment:
typo?

Reply:
No, this was correct. The delivery mech cannot have a bigger delivery rate than requester agreed for. `mechDelivery.deliveryRate` is the max delivery rate that requester chose when selecting the priority mech.